src/api/spec/models/kiwi/package_group_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Kiwi::PackageGroup, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { is_expected.to define_enum_for(:kiwi_type).with(%i[bootstrap delete docker image iso lxc oem pxe split testsuite vmx]) }
```

Comment:
do we want to check the types? I am think about having to modify the test if we want to add a new type :thinking: